File: Task/JortSort/Python/jortsort.py
>>> def jortsort(sequence):
	return list(sequence) == sorted(sequence)
>>> for data in [(1,2,4,3), (14,6,8), ['a', 'c'], ['s', 'u', 'x'], 'CVGH', 'PQRST']:
	print(f'jortsort({repr(data)}) is {jortsort(data)}')
jortsort((1, 2, 4, 3)) is False
jortsort((14, 6, 8)) is False
jortsort(['a', 'c']) is True
jortsort(['s', 'u', 'x']) is True
jortsort('CVGH') is False
jortsort('PQRST') is True
>>>

File: Task/JortSort/PascalABC.NET/jortsort.pas
##
function jortsort<T>(Self: sequence of T): boolean;
extensionmethod;
begin
  result := Self.SequenceEqual(Self.sorted);
end;

(1..10).jortsort.println;
[1,2,3].jortsort.println;
'abcdefgc'.jortsort.Println;
